packages/shared/blockchain/btc-based/src/lib/nested/btc-based.tx.ts

```diff
@@ -147,18 +206,41 @@ export const btcBasedTransactions = (
     }
   }
 
+  const validateBalanceFromUTXO = async (
+    body: BtcFromUtxoTypes | LtcFromUtxoTypes,
+    utxos: BtcUTXO[] | LtcUTXO[]
+  ): Promise<boolean> => {
+    // get all UTXOs balances
+    const totalBalance = utxos.reduce((sum, u) => sum + u.value, 0)
```

Comment:
I would wrap u.value in bigNumber to avoid string instead of number - in general, everywhere